pylintrc

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 [MESSAGES CONTROL]
-disable=missing-docstring
+disable=missing-docstring, too-many-locals, anomalous-backslash-in-string, dangerous-default-value, attribute-defined-outside-init, import-error, broad-except, redefined-variable-type, superfluous-parens, too-few-public-methods
```

Comment:
I would not disable the too-many-locals or the too-few-public-methods either. Maybe you could create a [DESIGN] section in this `pylintrc`, where you can set the max-locals to 20 and the min-public-methods to 0.